SQLDB/Queries/insertData.sql

```diff
@@ -0,0 +1,26 @@
+INSERT INTO dbo.Users(username, userPw)
+VALUES
+	('Shiv', 'ShivPW'),
+	('Benji', 'BenjiPW'),
+	('Frank', 'FrankPW'),
+	('Frida', 'FridaPW');
+
+INSERT INTO dbo.Books(isbn, title, authors)
+VALUES
+	('978-1407104027', 'Invisible City', 'MG Harris'),
+	('978-1406360202', 'Point Blanc', 'Anthony Horowitz'),
+	('978-0747532743', 'Harry Potter and the Philosopher''s Stone', 'J. K. Rowling'),
+	('978-0141346809', 'Percy Jackson and the Lightning Thief', 'Rick Riordan');
+
+INSERT INTO dbo.LibraryBooks(isbn)
+VALUES
+	('978-1407104027'),
+	('978-1407104027'),
+	('978-1407104027'),
+	('978-1406360202'),
+	('978-0747532743'),
+	('978-0141346809');
+
+INSERT INTO dbo.Loans(userId, bookId, dueDate)
+VALUES
+	(1, 1, '2020-10-28')
```

Comment:
To make sure these ids are correct you can set the ids when creating the users and books. To do that you need to set `IDENTITY_INSERT` to `ON` for the table you're editing, e.g.
```
SET IDENTITY_INSERT MyTable ON
INSERT INTO MyTable (Id, Name)
  VALUES (1, "Anna"), (2, "Bob")
SET IDENTITY_INSERT MyTable OFF
```
🙂